crates/globset/Cargo.toml

```diff
@@ -21,6 +21,7 @@ bench = false
 
 [dependencies]
 aho-corasick = "1.1.1"
+arbitrary = { version = "1.3.2", optional = true, features = ["derive"] }
```

Comment:
I don't like implicitly depending on crate names for features. Could you explicitly add an `arbitrary` feature? Example: https://github.com/rust-lang/regex/blob/0c0990399270277832fbb5b91a1fa118e6f63dba/regex-syntax/Cargo.toml#L18

Reply:
Please also add docs for this feature in the crate docs, e.g., https://docs.rs/regex-syntax/latest/regex_syntax/#crate-features

Reply:
Would you be ok with me adding a `fuzz` folder with a simple fuzz test?

Reply:
Probably fine. But it needs to be tested in CI.

Reply:
Thanks @BurntSushi. I've setup a `fuzz` directory and added a `fuzz_testing` action to ci.yml. Let me know if you want to adjust the duration for the fuzz testing as this can be modified. I've also addressed the other comments above.